java/client/test/org/openqa/selenium/chrome/ChromePrintCommandTest.java

```diff
@@ -0,0 +1,79 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.chrome;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Before;
+import org.openqa.selenium.printoptions.PageSize;
+import org.openqa.selenium.printoptions.PrintOptions;
+import org.openqa.selenium.testing.JUnit4TestBase;
+
+public class ChromePrintCommandTest extends JUnit4TestBase {
```

Comment:
Why is this chrome only? We expect Edge, Firefox and Safari to all support this functionality.